dart/tool/service.md

```diff
@@ -1876,13 +1879,11 @@ class Instance extends Object {
   //   Closure
   @Function closureFunction [optional];
 
-  // TODO(devoncarew): this can return an InstanceRef
-  //
   // The context associated with a Closure instance.
   //
   // Provided for instance kinds:
   //   Closure
-  //@Context closureContext [optional];
+  @Context closureContext [optional];
```

Comment:
@bkonyi - here, I have seen the VM return InstanceRefs instead of ContextRefs. This causes problems for us at deserialization time as we can't assign an InstanceRef to a ContextRef. I've temporarily commented the field out of the spec so we don't try and deserialize it, but we should probably resolve this in the VM.